plugins/react/frontend/components/form/ComboBox/index.js

```diff
@@ -325,48 +277,15 @@ export default class ComboBox extends Component {
   }
 
   render() {
-    let inputStyle = {
-      ...style.style,
-      ...this.props.style,
-    };
-    const wrapperStyle = {
-      ...style.wrapperStyle,
-      ...this.props.wrapperStyle,
-    };
-    const menuStyle = {
-      ...style.menuStyle,
-      ...this.props.menuStyle,
-    };
+    const menuStyle = {};
 
     const tabIndex = this.props.tabIndex ? this.props.tabIndex : '0';
 
-    if (this.props.disabled) {
-      inputStyle = {
-        ...inputStyle,
-        ...style.disabledStyle,
-        ...this.props.disabledStyle,
-      };
-    }
-
-    // todo: Currently there are no different hover styles for caret, like select they are probably not really needed.
-    let caretStyle;
-    if (this.props.disabled) {
-      caretStyle = {
-        ...style.caretToOpenStyle,
-        ...this.props.caretToOpenStyle,
-        ...style.disabledCaretToOpenStyle,
-        ...this.props.disabledCaretToOpenStyle,
-      };
-    } else if (this.state.isOpen) {
-      caretStyle = {
-        ...style.caretToCloseStyle,
-        ...this.props.caretToCloseStyle,
-      };
+    let caretClass;
+    if (this.state.isOpen) {
+      caretClass = styles.caretToCloseStyle;
     } else {
-      caretStyle = {
-        ...style.caretToOpenStyle,
-        ...this.props.caretToOpenStyle,
-      };
+      caretClass = styles.caretToOpenStyle;
     }
 
     const computedMenuStyle = (this.state.isOpen && !this.props.disabled && this.state.filteredOptions && this.state.filteredOptions.length > 0) ? menuStyle : { display: 'none' };
```

Comment:
Instead of using inline styles for the computedMenuStyle this could be menuClassname, but that's super minor …